### PR TITLE
Change icon for NPC achievement

### LINKED ISSUE #8611: Replace Current NPC Badge image
The current image for the NPC badge is the same as the one for Warrior-Ultimate Gear. An artisan has created a new image for this achievement- it can be found here: 

https://trello.com/c/Z65N9dNA/13-updated-npc-badge-image

### PR DESCRIPTION
**Pull request by librarianmage**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes #8611 

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

File: website/common/script/content/achievements.js [hunk 1]
```diff
@@ -107,7 +107,7 @@ let specialAchievs = {
     textKey: 'contribText',
   },
   npc: {
-    icon: 'achievement-ultimate-warrior',
```
Comment (paglias) [inline website/common/script/content/achievements.js:110]: is this icon still used somewhere else? If not it should be removed

**Review by paglias — CHANGES_REQUESTED** [review]

**Comment by librarianmage** [comment]
It is used in the ultimate warrior achievement, I believe.

File: website/common/script/content/achievements.js (hunk 1 above)
Comment (khipkin) [reply website/common/script/content/achievements.js:110]: it's used for the ultimate warrior achievement.

**Review by paglias — APPROVED** [review]

**Comment by paglias** [comment]
ok, another question: i saw in the folder all images have both a 2x and a normal resolution image while this PR only adds the 2x image. Isn't the normal one missing?

**Comment by librarianmage** [comment]
I was not provided a non-2x version.

**Comment by paglias** [comment]
Cc @beffymaroo

**Comment by librarianmage** [comment]
@paglias As I understand it, sprites are created in 1x scale and then are scaled up.

**Review by awitherow — APPROVED** [review]
Considering it is the only image attached to the issue, perhaps we should get this merged...

**Comment by paglias** [comment]
I'm just unsure this will work properly without the 1x version. Checking
with @SabreCat

Il 10 Apr 2017 00:05, "Austin Witherow" <notifications@github.com> ha
scritto:

**Comment by SabreCat** [comment]
I'm not aware of anyplace we still use the 1x version of achievements since the overhaul of the cheevos page @khipkin did a while back. We should be OK with just the 2x version!

**Comment by khipkin** [comment]
Yeah I can't think of anywhere still using the 1x versions. I think now they're just legacy. 😛

**Comment by khipkin** [comment]
#7904 for reference

**Comment by SabreCat** [comment]
@MathWhiz Merged, thank you! I will add this to your growing Tier 6 progress list.

**Comment by librarianmage** [comment]
😄

**Pull request merged**